zulip_bots/zulip_bots/bots/flock/flock.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import requests
+from typing import Any, Dict
+from requests.exceptions import ConnectionError
+
+USERS_LIST_URL = 'https://api.flock.co/v1/roster.listContacts'
+SEND_MESSAGE_URL = 'https://api.flock.co/v1/chat.sendMessage'
+
+help_message = '''
+You can send messages to any Flock user associated with your account from Zulip.
+*Syntax*: **@botname to: message** where `to` is **firstName** of recipient.
+'''
+
+# Matches the recipient name provided by user with list of users in his contacts.
+# If matches, returns the matched User's ID
+def find_recipient(res: str, to: str) -> str:
+    for i in range(0, len(res)):
```

Comment:
You can say this:

~~~ py
for obj in res:
    if to == obj['firstName']
        return res['id']
~~~

It will implicitly return `None`, which you can check in the caller.